cmd/influxd/launcher/launcher.go

```diff
@@ -977,7 +977,10 @@ func (m *Launcher) run(ctx context.Context, opts *InfluxdOpts) (err error) {
 	replicationSvc := replications.NewService()
 	replicationServer := replicationTransport.NewReplicationHandler(
 		m.log.With(zap.String("handler", "replications")),
```

Comment:
Is this log ever used in the replication handler?

Reply:
Yes, it gets passed to `kithttp.NewAPI(kithttp.WithLog(log))`

Reply:
Ah yeah, I was confusing the hander with the service, since the log isn't used in the service.